src/jobs/run_codacy_plugins_test.yml

```diff
@@ -0,0 +1,36 @@
+description: "Runs codacy-plugins-test on a tool"
+
+executor: machine
+
+parameters:
+  codacy_plugins_test_version:
+    type: string
+    description: "codacy-plugins-test version"
+
+environment:
+  CODACY_PLUGINS_TEST_VERSION: << parameters.codacy_plugins_test_version >>
+
+steps:
+  - attach_workspace:
+      at: ~/
+  - restore_cache:
+      key: codacy-plugins-test-cache-$CODACY_PLUGINS_TEST_VERSION
+  - attach_workspace:
+      at: ~/
+  - run:
+      command: |
+        mkdir -p codacy-plugins-test
+        cd codacy-plugins-test
+        FILENAME=codacy-plugins-test-$CODACY_PLUGINS_TEST_VERSION
+        LINK="https://bintray.com/codacy/Binaries/download_file?file_path=$CODACY_PLUGINS_TEST_VERSION%2Fcodacy-plugins-test-linux"
+        if [ ! -f "$FILENAME" ]; then
+          wget $LINK -O $FILENAME
```

Comment:
In most linux distros, `wget` doesn't come with the core packages. My suggestion is to use `curl` instead, but if it comes with circle ci `machine`, I guess its totally fine.

Reply:
I remember Circle CI `machine` has `wget` but no `curl`. That's why I chose it.

Reply:
Pretty sure it also have `curl`, at least according to [this](https://circleci.com/docs/2.0/circleci-images/#pre-installed-tools) . But nevermind, `wget` is fine here.